script.js

```diff
@@ -0,0 +1,45 @@
+const form = document.querySelector('form');
+const section = document.querySelector('section');
+const books = JSON.parse(localStorage.getItem('Books'));
+function storeBooks() {
+  localStorage.setItem('Books', JSON.stringify(books));
+}
```

Comment:
- [ ] While starting the app the user can be to add a new book because the localStorage currently returns nothing, make sure to return an empty array which will allow you to push a new book.

```suggestion
const books = JSON.parse(localStorage.getItem('Books')) || [];
function storeBooks() {
  localStorage.setItem('Books', JSON.stringify(books));
}
```